pkg/types/ccipocr3/generic_types.go

```diff
@@ -83,9 +83,25 @@ func (c ChainSelector) String() string {
 }
 
 type CCIPMsg struct {
```

Comment:
Can we get rid of these and use the protobuf versions for everything?

Reply:
That would be nice, though the reason I'm somewhat hesitant is that these types are kind of used for different reasons: one set of types is for the OCR plugin to express information (right now marshaling to JSON) and one is to communicate w/ gRPC implementations of various interfaces. I wonder if these may diverge at some point or not, I'm not sure right now if they will or won't.

If we're confident they will always be in sync then I'm in favor of just using the protobuf generated files and axing the duplication

Reply:
If we have good tooling for it (i.e when we log observations, we don't log the protobuf but a human-readable version) it'd be a win

Reply:
I like this idea. I have same concern with Makram but we can decouple it after we reach that point.

Reply:
After trying to write the code though it starts getting complicated.
I added a ticket to properly investigate this approach.